eng/service.proj

```diff
@@ -71,4 +76,18 @@
             SkipNonexistentProjects="false"
             SkipNonexistentTargets="true" />
   </Target>
+  <Target Name="ProjectDependsOn">
+    <MSBuild Projects="@(ProjectReference)"
+            Targets="ProjectDependsOn"
+            BuildInParallel="$(BuildInParallel)"
+            SkipNonexistentProjects="false"
+            SkipNonexistentTargets="true">
+      <Output ItemName="ProjectsToInclude" TaskParameter="TargetOutputs"/>
+    </MSBuild>
+    <RemoveDuplicates Inputs="@(ProjectsToInclude->Replace($(RepoRoot), '.'))">
```

Comment:
I suspect this `.` isn't enough because you likely also need the beginning slash.

Reply:
Here is what output looks like:
```
$(RepoRoot)sdk\agrifood\Azure.Verticals.AgriFood.Farming\tests\Azure.Verticals.AgriFood.Farming.Tests.csproj
$(RepoRoot)sdk\alertsmanagement\Azure.ResourceManager.AlertsManagement\tests\Azure.ResourceManager.AlertsManagement.Tests.csproj
$(RepoRoot)sdk\anomalydetector\Azure.AI.AnomalyDetector\tests\Azure.AI.AnomalyDetector.Tests.csproj
```